contracts/IAM.sol

```diff
@@ -27,66 +48,120 @@ contract IAM {
+     * @dev Can only be called by the owner of IAM contract. Verification of beneficiary is done off the chain
+     * @param organisation The address of an Ethereum account representing the newly verified beneficiary to add
+     */
+    function add(address organisation) public ownerOnly {
+        require(orgStatus[organisation] == Status.NONE, "Organisation address already exists");
+        orgStatus[organisation] = Status.VERIFIED;
+        orgList.push(organisation);
+        emit AddVerifiedOrg(organisation);
     }
 
+    /**
+     * @notice Sets a beneficiary to have the 'Verified' status
+     * @dev Can only be called by the owner of IAM contract and beneficiary must not have 'NONE' status in orgStatus list
+     * @param organisation The address of an Ethereum account representing the beneficiary to set as 'Verified'
+     */
     function setVerified(address organisation) public ownerOnly registeredOnly(organisation) {
-        orgStatus[organisation] = status.VERIFIED;
+        orgStatus[organisation] = Status.VERIFIED;
         if (dateOfDistrust[organisation] != 0) {
             delete dateOfDistrust[organisation];
         }
-        emit orgVerified(organisation);
+        emit OrgVerified(organisation);
     }
 
+    /**
+     * @notice Sets a beneficiary to have the 'Locked' status
+     * @dev Can only be called by the owner of IAM contract and beneficiary must not have 'NONE' status in orgStatus list
+     * @param organisation The address of an Ethereum account representing the beneficiary to set as 'Locked'
+     */
     function setLocked(address organisation) public ownerOnly registeredOnly(organisation) {
-        orgStatus[organisation] = status.LOCK;
+        orgStatus[organisation] = Status.LOCK;
         if (dateOfDistrust[organisation] != 0) {
             delete dateOfDistrust[organisation];
         }
-        emit orgLocked(organisation);
+        emit OrgLocked(organisation);
     }
 
+    /**
+     * @notice Sets a beneficiary to have the 'Distrust' status
+     * @dev Can only be called by the owner of IAM contract and beneficiary must not have 'NONE' status in orgStatus list
+     * @param organisation The address of an Ethereum account representing the beneficiary to set as 'Distrust'
+     */
     function setDistrust(address organisation) public ownerOnly registeredOnly(organisation) {
-        orgStatus[organisation] = status.DISTRUST;
+        orgStatus[organisation] = Status.DISTRUST;
         dateOfDistrust[organisation] = block.timestamp;
-        emit orgDistrust(organisation);
+        emit OrgDistrust(organisation);
     }
 
-    // --- FUNCTIONS ---
-    // adds a verified organisation
-    function add(address organisation) public ownerOnly {
-        require(orgStatus[organisation] == status.NONE, "Organisation address already exists");
-        orgStatus[organisation] = status.VERIFIED;
-        orgList.push(organisation);
-        emit addVerifiedOrg(organisation);
+    /**
```

Comment:
Likewise as above.

Reply:
Updated in 8c9a46e. I've added a short comment next to the `dateOfDistrust` state variable to clarify that the time is stored in Unix time format since that info was in the `getRefundPeriod` function docstring